### DISCUSSION TITLE
Many issues

### DISCUSSION THREAD

**Discussion by Ahmadmansoor** [CommunityToolkit/Maui#465] [General]
after I update maui and update CommunityToolkit.Mvvm to 8.0.0-preview4
a lot of issues happen like on Android like this code:
`<Frame Padding="2" Margin="0,10,0,10" BorderColor="Blue" BackgroundColor="CadetBlue"> <Label Text="{Binding Title,Mode=TwoWay}" VerticalTextAlignment="Center" HorizontalOptions="Center" HorizontalTextAlignment="Center" LineBreakMode="WordWrap" HeightRequest="70" VerticalOptions="CenterAndExpand" TextColor="DarkKhaki" /> </Frame>`
I found each word of the title is in a new line not on the same line ( not happen on iOS)
and the behavior of Maui is not stable when debugging.
Does anybody have the same issues ?????

**Comment by Sergio0694** [comment]
This is unrelated from the MVVM Toolkit. Not really sure it has anything to do with the MAUI Toolkit either.
@brminnick feel free to close if just out of scope, or possibly forward this to the MAUI repo if you have access there 🙂

**Reply by bijington** [reply]
No if anything this should be closed and a discussion opened on the .NET MAUI repo over at: https://github.com/dotnet/maui

@Ahmadmansoor Are you able to open a discussion or issue there? You will likely need to provide some extra detail to aid their understanding of the issue though

**Comment by CliffAgius** [comment]
Closed as answered.
